addOns/replacer/src/main/javahelp/org/zaproxy/zap/extension/replacer/resources/help/contents/replacer.html

```diff
@@ -17,6 +17,10 @@ <H1>Replacer</H1>
 <h3>Description</h3>
 A description that allows you to identify the rule.
 
+<h3>URL</h3>
+A regular expression to match the URL of the message, if empty the rule applies to all messages.<br>
+For example, allows to restrict the rules to certain domains when providing authentication credentials.
+
```

Comment:
Not necessarily for this PR, but should we have an option to apply the rules to a Context? That way users wouldn't have to define the same regexes twice (potentially).

Reply:
That makes sense, could be similar to alert filters, which can be defined in the contexts themselves.

Reply:
Ya exactly